covid19_sfbayarea/data/hospitals.py

```diff
@@ -26,14 +26,23 @@
 SERIES_NAME = "Hospitalization"
 BAYPD_META = "This data was pulled from the data.ca.gov CKAN Data API"
 
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(
+    format="%(asctime)s %(message)s",
+    level=logging.INFO
+)
 
 
 def get_county(county: str) -> Dict:
     """Return data just for the selected county. Include field notes.
     This is basically an alias for `get_timeseries()` provided for consistency.
     """
-    data = get_timeseries(county)
+    if county != "all":
+        clean_county = county.replace("_", " ").title()
```

Comment:
Ha! I hadn’t even seen this, but just merged basically the same code in `covid19_sfbayarea.utils.friendly_county()`. We should call that function here instead, so we get shared logic everywhere. https://github.com/sfbrigade/data-covid19-sfbayarea/blob/6b747d71f696212bbac324c880fa51f929725d14/covid19_sfbayarea/utils.py#L1-L6